### DISCUSSION TITLE
Update a second table with data selected from first table

### DISCUSSION THREAD

**Discussion by bobpeers** [Budibase/budibase#10408] [Help]
This use case seems fairly simple but it's hard to see how this works in Budibase.

Use case:
Get orders and items from an SQL server then let the user select an order and some selected items off the order (for example show eCommerce order numbers and then let them check multiple items off the order to return)

After selecting the order and items write this data to **another** table along with a timestmp for example.

It looks to me like the form builder expects to write to the same table, not another table. How could I achieve this?

thanks,
Bob

**Comment by melohagan** [comment]
Hi @bobpeers 

Is this your use-case here? https://docs.budibase.com/docs/filter-by-relationships

**Reply by bobpeers** [reply]
@melohagan Thanks, I'll check it out to see if it's what I need
